cmd/dm-master/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
+approvers:
+  - sig-approvers-dm
+labels:
+  - area/dm
```

Comment:
Also the bot will add the label `area/dm` automatically when changes hit this scope.